src/main/java/emissary/admin/PlaceStarter.java

```diff
@@ -161,7 +166,7 @@ public static IServiceProviderPlace alreadyExists(final String theLocation) {
         final String luStr = theLocation.substring(theLocation.indexOf("//"));
         try {
             final IServiceProviderPlace thePlace = (IServiceProviderPlace) Namespace.lookup(luStr);
-            logger.debug(theLocation + " already running on " + thePlaceHost);
+            logger.debug("{} already running on {}", theLocation, thePlaceHost);
             return thePlace;
         } catch (NamespaceException nse) {
```

Comment:
Should these two catches be combined. Or should we log something?

Reply:
I don't know why we're catching Throwable here. It's on my list to look into. Agreed, we either need to combine them or remove the Throwable catch. I'm in favor of doing the latter, but I thought it may be a little out of scope for this pr. It shows up in the sonarlint report, so it'll be on my next iteration to cleanup.